content/docs/command-reference/commit.md

```diff
@@ -71,6 +71,10 @@ force-update the `dvc.lock` or `.dvc` files and save data to cache. They are
 still useful, but keep in mind that DVC can't guarantee reproducibility in those
 cases.
 
+We can [add outputs to a stage](/docs/user-guide/how-to/add-output-to-stage)
+without re-running it, and `dvc commit` is used to save the newly specified
+output to the cache and update `dvc.lock`.
+
```

Comment:
Looks like we're just appending the notes to the end of the Description. Let's read and understanding the context first to find the best place and manner to insert this info. Hint: there's actually a bullet list of scenarios to use this command in this ref.

Reply:
The explanation is missing a key piece of info: it's about EXISTING stages (that have already been executed). Let's also use the term "execute" instead of "run" here BTW, please. Also, no need to explain what commit does since that is explained in detail in this reference. Again, let's try to understand the context, read the doc, and make sure the text being inserted makes sense when reading it as a whole.